### PR TITLE
ci(gitlab): fix notify stages

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by douglasduteil** [comment]
Works @lionelB 
On merge ?

**Comment by lionelB** [comment]
GO!

File: .gitlab-ci/stages/notify.yml [hunk 1]
```diff
@@ -10,39 +10,76 @@ include:
...
+.notify_dev:
+  environment:
+    name: fabrique-dev
   except:
     variables:
+      # Don't run when deploying in production an existing image
+      - $PRODUCTION
       # Don't run when running e2e tests
       - $E2E_TEST
+      # Don't register a base image for release commit
```
Comment (lionelB) [inline .gitlab-ci/stages/notify.yml:36]: weird comment, no ?

Comment (douglasduteil) [reply .gitlab-ci/stages/notify.yml:36]: same as https://github.com/SocialGouv/code-du-travail-numerique/blob/master/.gitlab-ci/stages/deploy.yml#L68

Comment (lionelB) [reply .gitlab-ci/stages/notify.yml:36]: weird too :-)

**Pull request merged**